engines/griffon/draw.cpp

```diff
@@ -0,0 +1,1230 @@
+					for (int ff = 0; ff <= 2; ff++) {
+						if (_npcInfo[i].hp > 10 * ff * 20) {
+							rcSrc.left = 16 * (int)(RND() * 2);
+							rcSrc.top = 80;
+							rcSrc.setWidth(16);
+							rcSrc.setHeight(16);
+
+							rcDest.left = _npcInfo[i].bodysection[10 * ff].x - 8;
+							rcDest.top = _npcInfo[i].bodysection[10 * ff].y - 8;
+
+							int x = 192 + ((int)(_itemyloc + ff * 5) % 3) * 64;
+							if (x > 255)
+								x = 255;
+							_spellImg->setAlpha(x, true);
+							_spellImg->blit(*_videoBuffer, rcDest.left, rcDest.top, Graphics::FLIP_NONE, &rcSrc);
+
+							for (int f = 1; f <= 8; f++) {
+								rcSrc.left = 16 * (int)(RND() * 2);
+								rcSrc.top = 80;
+								rcSrc.setWidth(16);
+								rcSrc.setHeight(16);
+
+								rcDest.left = _npcInfo[i].bodysection[ff * 10 + f].x - 8 + (int)(RND() * 3) - 1;
+								rcDest.top = _npcInfo[i].bodysection[ff * 10 + f].y - 8 + (int)(RND() * 3) - 1;
+
+								x = 192 + f % 3 * 64;
+								if (x > 255)
+									x = 255;
+								_spellImg->setAlpha(x, true);
+								_spellImg->blit(*_videoBuffer, rcDest.left, rcDest.top, Graphics::FLIP_NONE, &rcSrc);
+							}
+
+							rcSrc.left = 0;
+							rcSrc.top = 0;
+							rcSrc.setWidth(42);
+							rcSrc.setHeight(36);
+
+							rcDest.left = _npcInfo[i].bodysection[10 * ff + 9].x - 21;
+							rcDest.top = _npcInfo[i].bodysection[10 * ff + 9].y - 21;
+
+							_spellImg->setAlpha(192, true);
+							_anims[5]->blit(*_videoBuffer, rcDest.left, rcDest.top, Graphics::FLIP_NONE, &rcSrc);
+						}
+
+					}
+
+				}
+
+				// red dragon
+				if (_npcInfo[i].spriteset == kMonsterRedDragon) {
+					int cframe = _npcInfo[i].cframe;
+
+					rcSrc.left = (int)(cframe / 4) * 24;
+					rcSrc.top = wdir * 24;
+					rcSrc.setWidth(24);
+					rcSrc.setHeight(24);
+
+					rcDest.left = npx;
+					rcDest.top = npy;
+					rcDest.setWidth(24);
+					rcDest.setHeight(24);
+
+					if (_npcInfo[i].pause > _ticks && _npcInfo[i].shake < _ticks) {
+						_npcInfo[i].shake = _ticks + 50;
+						rcDest.left = rcDest.left + (int)(RND() * 3) - 1;
+						rcDest.top = rcDest.top + (int)(RND() * 3) - 1;
+					}
+
+					_anims[sprite]->blit(*_videoBuffer, rcDest.left, rcDest.top, Graphics::FLIP_NONE, &rcSrc);
+				}
+
+				// wizard
+				if (_npcInfo[i].spriteset == kMonsterPriest) {
+					// if(_npcinfo[i].attacking == 0) {
+					int cframe = _npcInfo[i].cframe;
+
+					rcSrc.left = (int)(cframe / 4) * 24;
+					rcSrc.top = wdir * 24;
+					rcSrc.setWidth(24);
+					rcSrc.setHeight(24);
+
+					rcDest.left = npx;
+					rcDest.top = npy;
+					rcDest.setWidth(24);
+					rcDest.setHeight(24);
+
+					if (_npcInfo[i].pause > _ticks && _npcInfo[i].shake < _ticks) {
+						_npcInfo[i].shake = _ticks + 50;
+						rcDest.left = rcDest.left + (int)(RND() * 3) - 1;
+						rcDest.top = rcDest.top + (int)(RND() * 3) - 1;
+					}
+					_anims[sprite]->blit(*_videoBuffer, rcDest.left, rcDest.top, Graphics::FLIP_NONE, &rcSrc);
+					// } else {
```

Comment:
Unless there's a particular need to keep the commented out code present, it'd be better to simply remove it; it'll still be in the history if it's ever needed. Otherwise, maybe add a comment as why the block is commented out, and why it might be needed again in the future

Reply:
Done.